about.html

```diff
@@ -0,0 +1,145 @@
+            <a href="./home.html"><img class="container" src="./Assets/event.png"/></a>
+            <ul class="nav-list container">
+                <li class="nav-item"><a href="./about.html"><h4>About</h4></a></li>
+                <li class="nav-item"><a href=#><h4>Program</h4></a></li>
+                <li class="nav-item"><a href=#><h4>Join</h4></a></li>
+                <li class="nav-item"><a href=><h4>Sponsor</h4></a></li>
+                <li class="nav-item"><a href=#><h4>News</h4></a></li>
+                <li class="nav-item" id="campaign"><a href=#><h4>CC Campaign</h4></a></li>
+            </ul>
+        </nav>
+        <header>
+            <div class="container">
+                <figure class="burger" alt="">
+                    <i class="fa fa-bars"></i>
+                </figure>
+                <div class="menu">
+                    <figure class="cancel" alt="">
+                        <i class="fa fa-times"></i>
+                    </figure>
+                    <ul class="menu-list">
+                        <li class="menu-item"><a href="./home.html"><h4>Home</h4></a></li>
+                        <li class="menu-item"><a href="#"><h4>About</h4></a></li>
+                        <li class="menu-item"><a href="#program"><h4>Program</h4></a></li>
+                        <li class="menu-item"><a href="#partner"><h4>Partners</h4></a></li>
+                    </ul>
+                </div>
+            </div>
+        </header>
+        <section id="front" class="container">
+            <div class="container" id="hello">
+                <h5>#Sadico</h5>
+                <h1>WELCOME TO
+                    <br>
+                    RAWAYANA'S LATIN  
+                    <br>
+                    AMERICA TOUR 2023
+                </h1>
+            </div>
+
+            <div class="container" id="info">
+                <p> 
+                    In 2022, Rawayana continued to tour internationally in support of their latest critically acclaimed album, 
+                    "Cuando Los Acéfalos Predominan", performing at major festivals.
+                    <br><br>
+                    This year, the band has released several singles like “Binikini” (feat. Danny Ocean), and “Dame Un Break”.
+                    <span class="show">
+                    As a prolific touring band, Rawayana is gearing up to support their new music with touring and festival 
+                    stops in Latin America, North America (over 30 cities in Q4), and Europe throughout 2023 and 2024.
+                    </span>
+                </p>
+            </div>
+            <div class="container" id="contact">
+                <p> 
+                    Please contact us via email for any further questions about the event!
+                    <br>
+                    <a href="mailto:sadicoficial@gmail.com">rawayanaoficial@gmail.com</a>
+                </p>
+            </div>
+        </section>
```

Comment:
- [ ] In this section on mobile screen, the `h1` heading should be the biggest text but in your mobile screen design the `h1` heading is smaller and the description is way bigger than it is in the behance design and also wrap you description div with borders and make sure you have enough margin on left and right as in the behance design.
your design
<img width="535" alt="Screenshot 2023-06-30 at 7 06 33 PM" src="https://github.com/Arch-Noize/capstone-portfolio/assets/96683843/aa39c577-5ef5-4a64-adf9-d4ceb9bc98f7">

behance design
<img width="517" alt="Screenshot 2023-06-30 at 7 06 23 PM" src="https://github.com/Arch-Noize/capstone-portfolio/assets/96683843/a5e23f8c-9938-4831-880d-50ab8ac2480a">